### DISCUSSION TITLE
Is it possible to change which sampler gets used with hires fix?

### DISCUSSION THREAD

**Discussion by Gazzoo-byte** [vladmandic/sdnext#557] [Q&A] [answered]
I notice that UniPC falls back to DDIM for it's hires fix, as far as I'm aware i2i isn't supported by UniPC as yet so I'm wondering is it possible to specify which sampler it uses instead? So for example, rather than going to DDIM as a fallback it could be configured to use DPM++ 2M Karras

**Comment by vladmandic** [comment]
valid request - can you create feature request for this so it doesn't fall off the radar?

**Reply by Gazzoo-byte** [reply]
https://github.com/vladmandic/automatic/issues/561 Doneski (I forgot there's a button for this in the discussion... whoops)

I may have a look into trying to write something for this, if it's not above my skill level you may see a pr from me with this

**Comment by vladmandic** [comment, marked as answer]
done :)

**Comment by cool-bigdogs-tshirt** [comment]
fyi unipc latent upscale was implemented, so this only applies to PLMS currently.

i did implement an override for XYZ charts though, and am considering implementing it as an option in the UI for latent upscaler. unipc latent upscale has been working well for me, but i've been wanting a way to manually select it for experimentation without using the XYZ script.

also, thank you vlad for asking the paper's author to look at my PR! until they actually review it I have no confidence that what it's doing actually makes any sense lol, but I'm still managing to get nice results.

**Reply by DarqueLilly** [reply]
did they fix this by chance? the unipc hiress issue?

**Reply by Gazzoo-byte** [reply]
Yes this has been fixed, you can now force a second pass sampler

**Reply by DarqueLilly** [reply]
so to make sure I understand you, You can force it to still hires unipc?

**Reply by vladmandic** [reply]
Yes
